1-org/README.md

```diff
@@ -20,51 +20,38 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Navigate into the repo `cd gcp-org` and change to a non prod branch `git checkout -b plan`
 1. Copy contents of foundation to new repo `cp -RT ../terraform-example-foundation/1-org/ .` (modify accordingly based on your current directory).
 1. Copy cloud build configuration files for terraform `cp ../terraform-example-foundation/build/cloudbuild-tf-* . ` (modify accordingly based on your current directory).
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```

Comment:
There is an duplication of the command. My sugestion:

Copy terraform wrapper script to the root of your new repository `cp ../terraform-example-foundation/build/tf-wrapper.sh . `(modify accordingly based on your current directory).

Reply:
We have this problem in a number of places, I think we should address this later and make sure we are not duplicating steps per build platform - I have raised this issue https://github.com/terraform-google-modules/terraform-example-foundation/issues/191